### PR TITLE
Add submission guidelines

### PR DESCRIPTION
**Pull request by BirgitBader**
@PMJP please give your feedback

### CODE REVIEW

**Comment by BirgitBader** [comment]
@RealMattMurdoch please proof-read

File: apps/app-submission-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+* Make sure your app performs as described and advertised.
+Apps that crash or have bugs will be rejected.
+* Be transparent with your app documentation and features.
+Apps that include undocumented or hidden features will be rejected.
+* We love to receive the final and production ready version of your app.
+We will not accept demo, trial or test versions.
+* Save your time and effort and don't duplicate an app that is already available in the Apps & Themes Store.
+We may reject it.
+* Concentrate on a reasonable and useful app.
+Apps that provide trick or mock a functionality will be rejected.
```
Comment (RealMattMurdoch) [inline apps/app-submission-guidelines.md]: I think this needs to be reviewed, it does not sound so good "provide trick".

File: apps/app-submission-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+# Metadata
+
+* We are keen on complete and meaningful text and descriptions in apps.
```
Comment (RealMattMurdoch) [inline apps/app-submission-guidelines.md]: We are attentive to complete and meaningful descriptions and text in all apps.

File: apps/app-submission-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+We will not accept apps that contain placeholder text.
```
Comment (RealMattMurdoch) [inline apps/app-submission-guidelines.md]: We will not accept apps that contain placeholder text or incomplete information.

File: apps/app-submission-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+Apps that perform contrary to this will be rejected.
+* We set a high value on security.
+We will reject apps that transmit viruses as well as files, code or programs that disrupt, damage or spy on our customer's data.
+* Protect customer data.
+We will not accept apps that require our customers to share e.g. their email address or date of birth in order to function.
+
+# Other
+
+* Our customers are our greatest asset and we respect them.
+We will not accept apps that are defamatory, insulting or mean.
```
Comment (RealMattMurdoch) [inline apps/app-submission-guidelines.md]: or rude (sounds better than mean)

File: apps/app-submission-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+* Apps must comply with all legal requirements in any country where they are made available to our customers.
+It is in the responsibility of the app developer to understand and conform to the respective local laws and regulations.
```
Comment (RealMattMurdoch) [inline apps/app-submission-guidelines.md]: It is in the responsibility of the app developer, to understand and conform to the respective local laws and regulations. Please send us an email to xx if you have any inquiries. 

*perhaps this is a good clause to add to save our butts.. we allow them to contact us and ask questions specific to this topic*

File: apps/app-submission-guidelines.md (hunk 1 above)
Comment (RealMattMurdoch) [inline apps/app-submission-guidelines.md]: here are some alternative suggestions:

Apps that do not function as expected, which trick the user or are in any form misleading shall be rejected.

Apps that trick the user or have hidden or mock functionality, shall be rejected.

**Pull request merged**